cli/src/main/java/com/devonfw/tools/ide/git/GitContextImpl.java

```diff
@@ -257,14 +259,35 @@ IdeContext getContext() {
   @Override
   public void verifyGitInstalled() {
 
-    this.context.findBashRequired();
-    Path git = Path.of("git");
-    Path binaryGitPath = this.context.getPath().findBinary(git);
-    if (git == binaryGitPath) {
-      String message = "Git is not installed on your computer but required by IDEasy. Please download and install git:\n"
-          + "https://git-scm.com/download/";
-      throw new CliException(message);
+    String bashBinary = this.context.findBashRequired();
+    String message = "Git is not installed on your computer but required by IDEasy. Please download and install git:\n"
+        + "https://git-scm.com/download/";
+
+    if (SystemInfoImpl.INSTANCE.isWindows()) {
+      Path bashPath = Path.of(bashBinary);
+      if (Files.exists(bashPath)) {
```

Comment:
see my comments above:
1. Already return `Path` instead of `String`
2. Return `null` if not exists on all platforms

Reply:
Adjusted.